app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
why do we need this? for the avatar? should the devise views be responsible for changing the avatar?

Reply:
i though having user profile edit in one place would be a bit cleaner and decided to reuse device edit view instead of creating new routers/controller with associated namespace and views. happy to do via separate namespace/outside devise

Reply:
we can delete this? and devise will just serve it's default view?